src/Common/src/Interop/Unix/System.Native/Interop.Poll.cs

```diff
@@ -20,9 +21,10 @@ internal enum PollEvents : short
 
         internal struct PollEvent
         {
-            internal int FileDescriptor;         // The file descriptor to poll
+            internal IntPtr FileDescriptor;      // The file descriptor to poll
```

Comment:
@nguerrera - do you think this one is necessary?  We don't get any of the benefits here, since we can't marshal using a SafeHandle.  And this change causes the size of PollEvent to increase and to require padding.

The only advantage we get is that it is consistent with the rest of the file descriptors.